spec/models/content_migration/course_copy_assignments_spec.rb

```diff
@@ -175,7 +181,11 @@
 
       new_assignment = @copy_to.assignments.where(migration_id: mig_id(@assignment)).first
       attrs.each do |attr|
-        expect(@assignment[attr]).to eq new_assignment[attr]
+        if @assignment[attr].class == Hash
```

Comment:
I don't think this block of code is needed.  Where you getting failure states without doing this?  The eq spec is smart enough to match hashes pretty well.